src/pages/getting_started/first_app.md

```diff
@@ -139,29 +143,44 @@ There are two sample flows listed below. Some developers may not have access to
     ✔ App initialization finished!
     ```
 
+1. Now your project is initialized! Go into the folder you just created, and you can see a number of files generated.
 
-1. Now your project is initialized! Go into the folder you just created, and you can see a number of files generated. 
-
-    ```
+    ```bash
     $ cd helloworld
     $ ls
-    README.md		src		app.config.yaml		package-lock.json	test
-    e2e			node_modules		package.json		web-src
+    README.md  src  app.config.yaml  package-lock.json test
+    e2e   node_modules  package.json  web-src
     ```
-    
+
 1. Note that you still can add/remove the extension points, back-end actions, SPA front-end or Github workflows from your application later by respectively using the `aio app <add|delete> ext`, `aio app <add|delete> action`, `aio app <add|delete> web-assets` and `aio app <add|delete> ci` commands within your application folder.
 
+1. Optionally, you can install an App Builder template using Template Registry. Discover available templates in the App Builder template registry with the following command:
+
+    ```bash
+    aio templates discover
+    ```
+
+1. Install the desired template with the following command:
+
+    ```bash
+    aio templates install <npm package name>
+    ```
+
+    The template's npm package will be downloaded and extracted, and Developer Console resources such as services and workspaces will be created and configured based on the template's **install.yaml** configuration file. `npm install` will run in the background to install the npm package.
+
 #### 4.1.2 Initializing an empty project
+
 1. In your Terminal, navigate to where you want to initialize your project and type in the following command in your Terminal:
 
-    ```
+    ```bash
     aio app init <app_name> --no-extensions
```

Comment:
with the cli with Template Registry API integration, `--no-extensions` does not make sense anymore and it has been changed to `--standalone-app`